ipxe/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+sudo zypper -n install cross-aarch64-gcc11 xz-devel
+
+test -d ipxe || git clone https://github.com/ipxe/ipxe.git
+pushd ipxe/src
+
+# this would compile the ipxe version using its internal driver implementations
+#make -j3 EMBED=../../boot.ipxe bin/ipxe.pxe bin-x86_64-efi/ipxe.efi
+#make -j3 EMBED=../../boot.ipxe CROSS=aarch64-suse-linux- bin-arm64-efi/ipxe.efi
+
+# undionly / snponly uses the network driver of the PXE / UEFI stack
+make -j3 EMBED=../../boot.ipxe bin/undionly.kpxe bin-x86_64-efi/snponly.efi
```

Comment:
Is the build system broken or why are you limiting this to 3 cores. If it is broken it would make more sense to limit it to 1 core. Otherwise I'd leave it out and not override MAKEFLAGS.

Reply:
netboot only has 2 cores